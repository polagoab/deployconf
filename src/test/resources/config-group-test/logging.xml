<?xml version="1.0" encoding="UTF-8"?>
<logging>
  <dir>@LOGDIR@</dir>
</logging>
