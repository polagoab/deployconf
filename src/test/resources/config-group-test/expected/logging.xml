<?xml version="1.0" encoding="UTF-8"?>
<logging>
  <dir>/etc</dir>
</logging>
